devlab/tests/generate_load.py

```diff
@@ -1032,6 +1035,17 @@ def break_image(self):
             self.migration_utils.execute_command_on_vm(
                 self.get_vagrant_vm_ip(), cmd, username='root', password='')
 
+    def create_network_with_segm_id(self):
+        self.dst_cloud = Prerequisites(
```

Comment:
I saw it. But it would be cross-dependant variable for functions.
If, for example, the create_tenant_wo_sec_group_on_dst function will be modified or moved to run after create_network_with_segm_id, then it will raise error.
I will add check if the variable exists - it will be the solution.